<def f='include/c++/4.8.2/bits/exception_ptr.h' l='115' ll='120' type='std::__exception_ptr::exception_ptr &amp; std::__exception_ptr::exception_ptr::operator=(std::__exception_ptr::exception_ptr &amp;&amp; __o)'/>
<use f='include/c++/4.8.2/future' l='371' u='c' c='_ZNSt13__future_base11_State_base16_M_break_promiseESt10unique_ptrINS_12_Result_baseENS2_8_DeleterEE'/>
